index.bs

```diff
@@ -778,58 +817,16 @@ spec: XHR; urlPrefix: https://xhr.spec.whatwg.org/
   present in a <a>suborigin policy</a> have the following effects:
 
   ### `'unsafe-postmessage-receive'` ### {#unsafe-postmessage-receive}
-  When a message is sent <i>to</i> a
-  frame with a `postMessage` `target` of a serialized physical origin, but not
-  a serialized suborigin, if the frame has an execution context with a
-  suborigin where the scheme, host, and port match the `target`, but it also
-  has a suborigin namespace, if `'unsafe-postmessage-receive'`</a> is set, it
-  will still receive the message.
-
-  <div class="example">
-  `https://example.com` runs a map API at `https://example.com/maps` which is
-  embedded by other websites. It provides a `postMessage()` API to place
-  markers on the map at locations the embedder chooses.
-
-  The developer would like to run `https://example.com/maps` in a suborigin
-  namespace "maps".  However, when embedders send messages to the embedded
-  frame, because they are legacy uses from before the use of suborigins, they
-  send essages with a `target` of `https://example.com`, <em>not</em>
-  `https-so://maps.example.com`. Since the developer would still like this
-  frame to be able to provide the API to these legacy embedders, it can serve
-  the frame with the `'unsafe-postmessage-receive'` directive, which will
-  allow the frame to receive messages on behalf of `https://example.com`, even
-  though it is at `https-so://maps.example.com`.
-  </div>
-
-  Issue: Should both postMessage opt-outs clarify that explicitly setting to
-  the serialized suborigin representation will still work? I.e., applications
-  that are aware of suborigins can use that if they want.
+  If `unsafe-postmessage-receive` is set, `MessageEvent`s sent to and from a
+  browsing context in the respective suborigin namespace will expose the
+  serialization of their physical origin via the `origin` attribute of the
+  `MessageEvent`s.
 
   ### `'unsafe-postmessage-send'` ### {#unsafe-postmessage-send}
-  When a message is sent <i>from</i> a
-  suborigin namespace with `'unsafe-postmessage-send'` set, the
-  `event.origin` value of the receiver should be set to the serialized
-  physical origin, not the serialized suborigin value. However, the
-  `event.suborigin` field should still be set to the name of the suborigin
-  namespace.
+  If `unsafe-postmessage-send` is set, the legacy `postMessage` method can
+  be used to communicate with the browsing context in the respective suborigin.
+  The missing suborigin parameter is then interpreted to be a wildcard.
```

Comment:
So, if `unsafe-postmessage-send` is set, why not set the event.origin property to physical origin and event.initiator.origin and event.initiator.suborigin to whatever would be usual.